python/src/trezorlib/debuglink.py

```diff
@@ -558,18 +557,12 @@ def wait_layout(self, wait_for_external_change: bool = False) -> LayoutContent:
         obj = self._call(
             messages.DebugLinkGetState(wait_layout=DebugWaitType.NEXT_LAYOUT)
         )
-        self.layout_dirty = True
         if isinstance(obj, messages.Failure):
             raise TrezorFailure(obj)
         return LayoutContent(obj.tokens)
 
     @contextmanager
-    def wait_for_layout_change(self) -> Iterator[LayoutContent]:
```

Comment:
i believe this removal isn't (and wasn't?) necessary -- you are asking for the default state, which will by default return the layout content, so you may as well populate it from the response, right?

(that said I'm surprised that this doesn't break any test code -- so kind of likely that nobody is using it?)

Reply:
It indeed seems that no one is using the returned layout - so I removed it, in order to simplify this method:
```
$ git grep wait_for_layout
python/src/trezorlib/debuglink.py:    def wait_for_layout_change(self) -> Iterator[None]:
tests/device_handler.py:        with self.debuglink().wait_for_layout_change():
```

WDYT?

Reply:
sure, why not